canonical-websites/www.ubuntu.com
# update /download/server to vbt

## Done

* update /download/server to vbt
* added p-strip—white, but I assume it was added in another PR? and what is the difference with .p-strip--light?
* fixed the ‘LTS’ abbreviation - and will do again in /download/desktop
* fixed the text-decoration on LTS in another PR
* the 'Ubuntu Server 16.04.1 LTS for OpenStack' section was new... so I changed the strip colors, but needs looking at

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:  [page](http://0.0.0.0:8001/download/server)
- compare to the [design](https://www.dropbox.com/search/work?path=%2F00_web_team%2F_ubuntu.com%2Fnew+projects%2Fvanilla+update%2Fpatterns+inventory&preview=download-server.png&qsid=01007770716940212773576207606415&query=download-server) and [demo](http://www.ubuntu.com-1638-download-server.demo.haus/download/server)

## Issue / Card

Fixes #1638
